php-checks/src/main/java/org/sonar/php/checks/ParameterSequenceCheck.java

```diff
@@ -48,24 +51,47 @@
   public void visitFunctionCall(FunctionCallTree tree) {
     // only check method and function calls expect constructors with more than 1 argument
     if (!tree.getParent().is(Kind.NEW_EXPRESSION) && tree.arguments().size() > 1) {
-      Symbol symbol = getDeclarationSymbol(tree);
-      if (symbol != null && symbol.declaration() != null && symbol.is(Symbol.Kind.FUNCTION)) {
-        checkParameterSequence(tree, (NameIdentifierTree) symbol.declaration());
+      if (tree.callee().is(Kind.NAMESPACE_NAME)) {
+        checkFunctionCall(tree);
+      } else {
+        checkMethodCall(tree);
       }
     }
 
     super.visitFunctionCall(tree);
   }
 
+  private void checkMethodCall(FunctionCallTree tree) {
+    Symbol symbol = getDeclarationSymbol(tree);
+    if (symbol != null && symbol.declaration() != null && symbol.is(Symbol.Kind.FUNCTION)) {
+      List<String> parameters = ((FunctionTree) symbol.declaration().getParent()).parameters().parameters().stream()
+        .map(e -> e.variableIdentifier().text())
+        .collect(Collectors.toList());
+
+      if (isWrongParameterSequence(tree, parameters)) {
+        newIssue(tree, String.format(MESSAGE, symbol.declaration().text()))
+          .secondary(symbol.declaration(), SECONDARY_MESSAGE);
+      }
+    }
+  }
+
+  private void checkFunctionCall(FunctionCallTree tree) {
+    FunctionSymbol symbol = Symbols.getFunction((NamespaceNameTree) tree.callee());
```

Comment:
As `visitFunctionCall` tests `tree.callee().is(Kind.NAMESPACE_NAME)`, I would prefer to have the cast `(NamespaceNameTree) tree.callee()` also in `visitFunctionCall` (that would probably require adding a new parameter to `checkFunctionCall`).